src/content/docs/en/guides/deploy/netlify.mdx

```diff
@@ -21,7 +21,7 @@ Your Astro project is a static site by default. You don’t need any extra confi
 
 ### Adapter for SSR
 
-To enable SSR in your Astro project and deploy on Netlify, including using Netlify's edge functions:
+To enable SSR in your Astro project and deploy on Netlify:
 
 Add [the Netlify adapter](/en/guides/integrations-guide/netlify/) to enable SSR in your Astro project with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
```

Comment:
I actually think we can probably collapse these two paragraphs as they basically repeat each other:

```suggestion
Add [the Netlify adapter](/en/guides/integrations-guide/netlify/) to enable SSR in your Astro project and deploy to Netlify with the following `astro add` command. This will install the adapter and make the appropriate changes to your `astro.config.mjs` file in one step.
```